src/components/Candidate.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import styled from 'styled-components';
+import { url } from './url';
+import axios from 'axios';
+
+const Candidate = ({ candidateName, voteCount, id, getCandidates }) => {
+  const getVote = async (id) => {
+    try {
+      const vote = await axios.get(`${url}/vote`, {
+        params: {
+          id: id,
+        },
+      });
+      getCandidates();
+    } catch (e) {
+      console.error(e);
+    }
+  };
+
+  const onClick = () => {
+    getVote(id);
+  };
```

Comment:
이벤트 핸들러는  `handle `접미사를 붙여서 네이밍 해주시면 좋을 것 같아요~
```suggestion
  const handleVoteCount= () => {
    updateVoteCount(id);
  };
```